plugins/ltac/g_tactic.mlg

```diff
@@ -703,7 +703,11 @@ GRAMMAR EXTEND Gram
       | IDENT "change"; c = conversion; cl = clause_dft_concl ->
 	  { let (oc, c) = c in
 	  let p,cl = merge_occurrences loc cl oc in
-	  TacAtom (CAst.make ~loc @@ TacChange (p,c,cl)) }
+          TacAtom (CAst.make ~loc @@ TacChange (true,p,c,cl)) }
```

Comment:
Could you take advantage of this change to translate the French comment just above?